volatility3/framework/constants/__init__.py

```diff
@@ -11,7 +11,6 @@
 import sys
 from typing import Optional, Callable
 
-import volatility3.framework.constants.linux
```

Comment:
Hmmmm, I think this is intended to ensure that the constants.linux namespace is available when just `volatility3.framework.constants` is imported.  Please could you re-instate it?

Reply:
Thank you, when I checked again, it was a meaningful import. 😅